server/index.ts

```diff
@@ -10,19 +9,19 @@ import {
   DEFAULT_ANALYSIS_DATA_SPARQL_QUERY,
   DEFAULT_REPORT_CATEGORIES,
   DEFAULT_REPORT_TYPES,
-} from './constants.js';
-
+} from './constants';
 
-process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
 const NODE_ENV = process.env.NODE_ENV;
-const PORT = process.env.PORT || 8000;
+const DISABLE_SSL = process.env.DISABLE_SSL;
+const PORT = Number(process.env.PORT) || 8000;
+// @ts-ignore
+process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = DISABLE_SSL;
```

Comment:
meant to be only for dev mode, wondering why an error raised due SSL cert, will investigate it (hopefully will not be in prod)